src/QsCompiler/Tests.Compiler/QirDriverGenerationTests.fs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.QsCompiler.Testing
+
+open Xunit
+open Xunit.Abstractions
+open Microsoft.Quantum.QsCompiler
+open Microsoft.Quantum.QsCompiler.BondSchemas.EntryPoint
+
+type QirDriverGenerationTests(output: ITestOutputHelper) =
+
+    [<Fact>]
+    member this.GenerateCommandLineArguments() =
```

Comment:
This approach seems difficult to scale. I would recommend to use an approach similar to the one used in `EntryPointSchemaTests`: https://github.com/microsoft/qsharp-compiler/blob/4b97de3ff0edf16969d918cf0bcf499697f9fea0/src/QsCompiler/Tests.Compiler/EntryPointSchemaTests.fs#L354

For the purposes of this PR, can you please add just this one test case in a setup similar to `EntryPointSchemaTests`? The reason for that is that this other [PR](https://github.com/microsoft/qsharp-compiler/pull/929) consolidates test cases in text files at `src\QsCompiler\Tests.Compiler\TestCases\QirEntryPointTests`. Once we have the test cases there, they can be used by multiple tests, including this one.

Reply:
Also, maybe this test could be moved to [this](https://github.com/microsoft/qsharp-compiler/blob/feature/azure-quantum-simulator/src/QsCompiler/Tests.Compiler/EntryPointGenerationTests.fs) other source file.

Reply:
This feedback will be addressed in a subsequent PR to unblock other on-going development in the feature branch.

Reply:
Discussed offline; will address in a separate PR